src/hotspot/share/runtime/lockStack.inline.hpp

```diff
@@ -26,14 +26,18 @@
 #ifndef SHARE_RUNTIME_LOCKSTACK_INLINE_HPP
 #define SHARE_RUNTIME_LOCKSTACK_INLINE_HPP
 
+#include "runtime/lockStack.hpp"
```

Comment:
Why was this pulled out first?

Reply:
I moved this include up because this is the mechanism we use to resolve circular dependencies between .inline.hpp files. If you take a look at most other .inline.hpp files you will see the same. I've written a section about this in the style guide:
https://github.com/openjdk/jdk/blob/master/doc/hotspot-style.md
> All .inline.hpp files should include their corresponding .hpp file as the first include line. Declarations needed by other files should be put in the .hpp file, and not in the .inline.hpp file. This rule exists to resolve problems with circular dependencies between .inline.hpp files.